.vscode/launch.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
nit: changes out of scope, this is one file I would leave out of this PR to reduce the PR size (it can be added in a separate PR if it helps)